scripts/generate-tts-languages

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+cd "$(dirname "$0")/.."
+
+
+if ! python3 -c "import hass_nabucasa" 2>/dev/null; then
+  coreversion=$(curl -sSL https://www.home-assistant.io/version.json | jq -r '.current_version')
+  corehassnabucasa=$(curl -sSL "https://raw.githubusercontent.com/home-assistant/core/refs/tags/${coreversion}/homeassistant/components/cloud/manifest.json" | jq -r '.requirements[]' | grep -E '^hass-nabucasa')
+  python3 -m pip install "${corehassnabucasa}" "josepy<2.0"
```

Comment:
`josepy` is not needed anymore, right?
```suggestion
  python3 -m pip install "${corehassnabucasa}"
```

Reply:
It is needed until core 2025.6.0 is released.